projects/plugins/jetpack/modules/subscriptions/subscriptions.js

```diff
@@ -0,0 +1,8 @@
+( function () {
+	window.addEventListener( 'load', function () {
+		var notificationElems = document.getElementsByClassName( 'jetpack-sub-notification' );
+		if ( notificationElems.length > 0 ) {
+			notificationElems[ 0 ].scrollIntoView();
+		}
```

Comment:
what do you think about adding a querySelector instead of the class name method?

```suggestion
		var notificationElement = document.querySelector ( '.jetpack-sub-notification' );
		if ( notificationElement ) {
			notificationElement.scrollIntoView();
		}
```

It returns the first occurrence, so you won't need to check by the array item.
Please take it as a simple suggestion and feel free to ignore it.

Reply:
It does look better! It is changed now!